build_resources.py

```diff
@@ -0,0 +1,80 @@
+import sys
+import subprocess
+import os
+
+
+# Set the Python environment variable. If 'PYTHONPATH' is not set, use the default path "/usr/bin/"
+# or you can override the Python environment path for a specific virtual environment
+PYTHON_ENV = os.getenv("PYTHONPATH", "/usr/bin/")
+
+# List of commands to build QT resources
+COMMANDS = [
+    [
+        "tk-build-qt-resources",
+        "-p",
+        PYTHON_ENV,
+        "-py",
+        "python/tank/authentication/ui",
+        "-q",
+        "python/tank/authentication/resources",
+        "-uf",
+        "login_dialog",
+        "-rf",
+        "resources",
+        "-i",
+        ".qt_abstraction",
+    ],
+    [
+        "tk-build-qt-resources",
+        "-p",
+        PYTHON_ENV,
+        "-py",
+        "python/tank/platform/qt",
+        "-q",
+        "python/tank/platform/qt",
+        "-uf",
+        "tank_dialog",
+        "item",
+        "busy_dialog",
+        "-ufn",
+        "ui_tank_dialog",
+        "ui_item",
+        "ui_busy_dialog",
```

Comment:
Why do we need to pass a list of `-uf` and `-ufn`?

I think it work better if we just add more items to `COMMANDS`, i.e. one for `tank_dialog`, other for `item`, and finally other for `busy_dialog`. We will make `COMMANDS` 😞  list bigger but keep the logic simpler for https://github.com/shotgunsoftware/tk-toolchain/pull/64

Let me know your thoughts.

Reply:
There are at most two more projects with this output .py file renaming. So I think the way I propose is better because compared to other repositories with many ui resources (like 10 or more), this is a better choice than a COMMAND for a single resource ui or rc.

Reply:
Fair enough. If you consider scalability then we're good to go 🚀